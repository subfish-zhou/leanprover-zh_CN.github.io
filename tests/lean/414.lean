macro_rules (kind := numLit)
 | `($n:numLit) => `("world")

#check 2

macro_rules
  | `($n:numLit) => `("hello")

#check 2

macro_rules (kind := numLit)
  | n => `("boo")

#check 2
