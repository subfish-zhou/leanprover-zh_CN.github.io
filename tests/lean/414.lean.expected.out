"world" : String
"hello" : String
"boo" : String
